tests/test_iib_client.py

```diff
@@ -269,6 +271,7 @@ def test_iib_client(
                 organization="org",
                 overwrite_from_index=True,
                 overwrite_from_index_token="str",
+                build_tags=["extra-tag1"],
```

Comment:
Shouldn't this be reflected in the result? We are comparing it to a model instance based on the fixture where `build_tags` is `["v4.5-2020-10-10"]`.